core/Rename.cpp

```diff
@@ -96,7 +104,59 @@ namespace olympia
             ev_rename_insts_.schedule();
         }
     }
+    void Rename::getAckFromROB_(const InstPtr & inst_ptr)
+    {
+        sparta_assert(inst_ptr->getStatus() == Inst::Status::RETIRED,
+                        "Get ROB Ack, but the inst hasn't retired yet!");
+
+        core_types::RegFile rf;
```

Comment:
Why not initialize this variable to `core_types::RegFile::RF_INTEGER` and remove the `else` clause?

Reply:
Addressed in [fbe7264](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/fbe7264a97b4aadea92369bdf93a544c3077551e)